docs/api/authentication.md

```diff
@@ -1,22 +1,32 @@
-# API Keys and authentication
+# Service accounts authentication
 
-API keys are used to control access to the PMM server components and resources. With an API key, you are authenticated to the PMM server, have access to PMM server components and resources, and perform various actions on them. You can use API keys as a replacement for basic authentication.
+!!! caution alert alert-warning "Important"
+    API keys are deprecated now. During update to version PMM v3 all of them you be converted into Service Accounts with Service Token. In such case API key persist same, but it is just covnerted into Service Token.
+
+Service accounts are used to control access to the PMM server components and resources. With an API key, you are authenticated to the PMM server, have access to PMM server components and resources, and perform various actions on them. You can use Service accounts as a replacement for basic authentication and API keys.
 
-##  Generate API keys
+Service account contains Service Tokens, which is direct replacement for API key.
 
-PMM uses the Grafana API keys for authentication. Following are the steps to generate the API keys:
+##  Generate Service Token 
+
+PMM uses the Grafana Service Token for authentication. Following are the steps to generate the Service Token:
```

Comment:
```suggestion
PMM uses the Grafana Service Account Tokens for authentication. A token is a generated random string that acts as an alternative to a API passwords.

Here's how to generate a Service Account Token:
```

Reply:
do we need to any any prerequisites here? What permissions are required for generating tokens?